_sources/Unit2-Using-Objects/topic-2-5-methods-return.rst

```diff
@@ -61,13 +61,13 @@ If a method is a **void method** and has **void** as its return type, like most
 Get Methods
 ------------
 
-Get methods return the value of instance variables, for example getWidth() and getHeight() to get the height and width for a Turtle object.
+Get methods return the value of instance variables, for example ``getWidth`` and ``getHeight`` to get the height and width for a ``Turtle`` object.
 
 In the previous lesson, we used some set methods with parameters to set the attributes of a turtle to different values, for example yertle.setColor(Color.red); or yertle.setWidth(50); Programmers create get and set methods for each attribute represented as an instance variable in a class to access and modify the value in that variable. The get methods always return back the value of that instance variable, and the set methods modify the value.
 
-When you use a get method, you need to save what it returns in a variable or use the value in some way for example by printing it out. The data type of the variable must match the data type of the return value of the method. You can find out the return type of a method in its documentation. It will be right before the method name, for example *int getWidth()* means getWidth will return an int (an integer number).
+When you use a get method, you need to assing what it returns in a variable or use the value in some way for example by printing it out. The data type of the variable must match the data type of the return value of the method. You can find out the return type of a method in its documentation. It will be right before the method name, for example ``int getWidth()`` means ``getWidth`` will return an ``int`` (an integer number).
```

Comment:
```suggestion
When you use a get method, you need to assign what it returns in a variable or use the value in some way for example by printing it out. The data type of the variable must match the data type of the return value of the method. You can find out the return type of a method in its documentation. It will be right before the method name, for example ``int getWidth()`` means ``getWidth`` will return an ``int`` (an integer number).
```

Reply:
Whoops! 🍑